userspace/otpilot/src/main.rs

```diff
@@ -259,9 +252,9 @@ async fn main() -> TockResult<()> {
     writeln!(console, "clock_frequency: {}", alarm::get().get_clock_frequency())?;
     let result = run();
     if result.is_ok() {
-        writeln!(console, "Returned OK.")?;
+        writeln!(console, "main: returning OK.")?;
     } else {
-        writeln!(console, "Returned error.")?;
+        writeln!(console, "main: returning error.")?;
```

Comment:
same thing here -- is there no value to printing the error that's being returned?

Reply:
Same as above - we don't have a printer for TockResult and it's covered by #227.

Reply:
:ack: